contrib/cf-remote/cf_remote/ssh.py

```diff
@@ -24,7 +25,12 @@ def connect(host, users=None):
     for user in users:
         try:
             log.debug("Attempting ssh: {}@{}".format(user, host))
-            c = fabric.Connection(host=host, user=user)
+            connect_kwargs = {}
+            key = os.getenv("CFREMOTE_SSH_KEY")
```

Comment:
I'd add an underscore to the name of the variable: `CF_REMOTE_SSH_KEY`.

Reply:
Fixed.